src/main/resources/assets/computercraft/lua/rom/programs/copy.lua

```diff
@@ -13,7 +13,11 @@ if #tFiles > 0 then
         if fs.isDir( sDest ) then
             fs.copy( sFile, fs.combine( sDest, fs.getName(sFile) ) )
         elseif #tFiles == 1 then
-            fs.copy( sFile, sDest )
+            if fs.exists( sDest ) then
+                 printError( "Destination exists" )
```

Comment:
SHOULD request of overwrite file instead of drop error and quit.
`fs.copy(sFile, sDest)` like use `--force` argument in POSIX/GNU `cp`.